build/buildconstants/params_calibnet.go

```diff
@@ -113,6 +110,15 @@ const UpgradeTeepHeight = 2520574
 // ramp behavior before mainnet upgrade.
 var UpgradeTuktukPowerRampDurationEpochs = uint64(builtin.EpochsInDay * 3)
 
+// 2025-03-25T23:00:00Z
+const UpgradeTeepHeight abi.ChainEpoch = 2520574
+
+var UpgradeTeepInitialFilReserved = wholeFIL(1_200_000_000) // FIP-0100: 300M -> 1.2B FIL
+
+// This epoch, 90 days after Teep is the completion of FIP-0100 where actors will start applying
+// the new daily fee to pre-Teep sectors being extended.
+var UpgradeTockHeight abi.ChainEpoch = UpgradeTeepHeight + builtin.EpochsInDay*90
```

Comment:
I think we landed on that the Tock upgrade should be 7 days after the Teep upgrade?
```suggestion
// This epoch, 7 days after Teep is the completion of FIP-0100 where actors will start applying
// the new daily fee to pre-Teep sectors being extended.
var UpgradeTockHeight abi.ChainEpoch = UpgradeTeepHeight + builtin.EpochsInDay*7
```

Reply:
good catch, thanks!